packages/components/src/VirtualizedList/CellDatetime/CellDatetime.component.js

```diff
@@ -45,24 +51,43 @@ export class CellDatetimeComponent extends React.Component {
 
 	render() {
 		const { cellData, columnData, t } = this.props;
+		const computedValue = computeValue(cellData, columnData, t);
+
 		const cell = (
 			<div className={classnames('cell-datetime-container', styles['cell-datetime-container'])}>
-				{computeValue(cellData, columnData, t)}
+				{computedValue}
 			</div>
 		);
 
 		if (columnData.mode === 'ago') {
+			let tooltipLabel = '';
+
+			if (columnData.timeZone) {
+				tooltipLabel = formatToTimeZone(cellData, columnData.pattern || DATE_TIME_FORMAT, {
+					timeZone: columnData.timeZone,
+				});
+			} else {
+				tooltipLabel = format(cellData, columnData.pattern || DATE_TIME_FORMAT);
+			}
+
 			return (
 				<TooltipTrigger
-					label={format(cellData, columnData.pattern || DATE_TIME_FORMAT)}
+					label={tooltipLabel}
 					tooltipPlacement={columnData.tooltipPlacement || 'bottom'}
 				>
 					{cell}
 				</TooltipTrigger>
 			);
 		}
 
-		return cell;
+		return (
+			<TooltipTrigger
+				label={columnData.tooltipLabel || computedValue}
+				tooltipPlacement={columnData.tooltipPlacement || 'bottom'}
+			>
+				{cell}
+			</TooltipTrigger>
+		);
```

Comment:
I think displaying tooltip with the same value as inside cell may be useful for cases when the date is not fully visible due to smaller column size
We have such case in TMC on new operations list (where we have 18 possible columns to display)

Reply:
Need UX approval.
cc @frassinier